docs/Release-Process.md

```diff
@@ -0,0 +1,36 @@
+# Release process for Swarm
+
+This document describes the steps required to perform a new release.
+
+## Pre release
+
+1. Make sure that the most recent builds are green and that smoke tests are passing on the cluster.
```

Comment:
I think the whole document is great. We should probably be a bit more explicit about what we mean with `smoke tests are passing on the cluster`. Perhaps we need a whole other document that describes what we consider a healthy commit ready to be released.

Reply:
agree. we can setup a baseline criteria of:

passing with no failures for 24 hours with:
- [ ] 10mb file
- [ ] 20mb file
- [ ] 40mb file
(and add more as we progress)